testing/multi-env-test-engine/src/main/java/org/projectnessie/junit/engine/MultiEnvDisplayNameGenerator.java

```diff
@@ -21,25 +21,28 @@
 public class MultiEnvDisplayNameGenerator implements DisplayNameGenerator {
 
   private final DisplayNameGenerator delegate;
-  private final String environment;
+  private final String environmentNames;
 
-  public MultiEnvDisplayNameGenerator(DisplayNameGenerator delegate, String environment) {
+  public MultiEnvDisplayNameGenerator(DisplayNameGenerator delegate, String environmentNames) {
     this.delegate = delegate;
-    this.environment = environment;
+    this.environmentNames = environmentNames;
```

Comment:
This particular class is meant to make make multi-env test distinguishable in tools that do not understand Junit5 `UniqueId` (i.e. those that deal with test class/method names only). `environmentIds` works in this case, but I still think the rename is unnecessary in this case as the old `environment` works just as well :)

Reply:
`1. Dimention` - SGTM (but let's wait a bit with renaming :) )  
`2. Dimention type` - I prefer segment (current) as it related to JUnit5 ID segments.
`3. Dimention elements` - Maybe `Category`? (in the statistical sense)

Reply:
Keeping `environment` as # 4 is fine with me. I don't think "category" should be used for elements, "category" is synonymous with dimension type / segment type. Also, I don't think we should use "segment" alone because a UniqueId segment is the pair of segment type + segment value.

It looks like we should choose

- One of (dimension | segment | category) for # 2
- Whether or not to use "type" as a suffix for # 2. I'd avoid "segment" alone, but could see "category" or "category type" working fine.
- One of (value | element | ID) for # 3


My vote at the moment is "dimension", "type", and "value":
```
public @interface MultiEnvDimensionType {}
public interface MultiEnvTestExtension extends Extension {
  List<String> allDimensionValues(ConfigurationParameters configuration);
  int segmentPriority()
}
```

For now, I've reverted these back to `environment` and will see what the rename would look like in practice.